### DISCUSSION TITLE
wr.athena.read_sql_query doesn't work

### DISCUSSION THREAD

**Discussion by gpandolfino** [aws/aws-sdk-pandas#1114] [Q&A]
Hi, I started using AWS data wrangler to execute queries on AWS Athena and use the results of them in my AWS Glue python shell job.
I see that exist wr.athena.read_sql_query to obtain what I need.

this is my sample code:
```
import sys
import os
import awswrangler as wr

os.environ['AWS_DEFAULT_REGION'] = 'eu-west-1'
databases = wr.catalog.databases()
print(databases)

query='select count(*) from staging_dim_channel'
print(query)
df_res = wr.athena.read_sql_query(sql=query, database="lsk2-target")
print(df_res)
print(f'DataScannedInBytes:            {df_res.query_metadata["Statistics"]["DataScannedInBytes"]}')
print(f'TotalExecutionTimeInMillis:    {df_res.query_metadata["Statistics"]["TotalExecutionTimeInMillis"]}')
print(f'QueryQueueTimeInMillis:        {df_res.query_metadata["Statistics"]["QueryQueueTimeInMillis"]}')
print(f'QueryPlanningTimeInMillis:     {df_res.query_metadata["Statistics"]["QueryPlanningTimeInMillis"]}')
print(f'ServiceProcessingTimeInMillis: {df_res.query_metadata["Statistics"]["ServiceProcessingTimeInMillis"]}')
```

I retrieve without problem the list of database (including the lsk2-target), but the read_sql_query go on error and I recieve:
`WaiterError: Waiter BucketExists failed: Max attempts exceeded`

Please, can you help me to understand where I am wrong?
thanks

**Comment by jaidisido** [comment]
Hi @gpandolfino, I believe a similar [issue](https://github.com/awslabs/aws-data-wrangler/issues/299) was raised in the past, did you have a look at the suggestion? Thanks
